p2p/src/types.rs

```diff
@@ -268,11 +287,24 @@ impl P2PConfig {
 		}
 	}
 
-	/// return peer_max_count
+	/// return total maximum peer count (incoming + outgoing)
```

Comment:
Do we still need this total max count anywhere?

Reply:
It was still used when broadcasting compact blocks and transactions for `Peers` but should be able to be removed in favor of making incoming and outgoing connections more explicit. Please see the unresolved question posted here https://github.com/mimblewimble/grin/pull/2985#issuecomment-521445528